### DISCUSSION TITLE
播放器底栏拥挤

### DISCUSSION THREAD

**Discussion by wulingyijiu** [the1812/Bilibili-Evolved#3780] [综合讨论 / General]
![image](https://user-images.githubusercontent.com/46135524/197708159-7bf7d6d8-daf2-4d4b-af9f-52575d84bb91.png)
当全屏或网页全屏时会显示”弹幕发送栏“而这会导致右侧的一些按钮变得拥挤，总之看着很难受
不知道有没有相关功能关闭”弹幕发送栏“

如果可以自定义显示播放器底栏功能按钮就更好了

**Comment by the1812** [comment]
> 如果可以自定义显示播放器底栏功能按钮就更好了

上古需求了 (#261)

**Reply by wulingyijiu** [reply]
![image](https://user-images.githubusercontent.com/46135524/197810455-3eb70bde-0682-4fbd-8e8c-dc405f03313f.png)
加进去后导入报错
![image](https://user-images.githubusercontent.com/46135524/197810527-9a74d802-0e41-4ba1-a911-dbcfc7c3f05e.png)

**Reply by the1812** [reply]
括号少了一个, `{}` 是要成对的

**Reply by wulingyijiu** [reply]
哦确实是少一个
现在好多了
![image](https://user-images.githubusercontent.com/46135524/197912623-f1388b74-b806-4d49-9d89-c0d3f7052bb1.png)
